web/docs/language/basic-elements.md

```diff
@@ -365,7 +369,33 @@ export const signUp = async (args, context) => {
 ```
 
 :::info
-You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp put middleware in place that takes care of hashing it before storing it to the database.
+You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp put middleware in place that takes care of hashing it before storing it to the database. An additional middleware also performs field validation.
+:::
+
+##### Customizing user entity validations
+
+To disable default validations, or add your own, you can do:
+```js
+const newUser = context.entities.User.create({
+  data: { email: 'some@email.com', password: 'this will be hashed!' },
+  _waspSkipDefaultValidations: true, // defaults to false
+  _waspCustomValidations: [
+    {
+      validates: 'password',
+      message: 'password must be present',
+      validator: password => !!password
+    },
+    {
+      validates: 'password',
+      message: 'password must be at least 8 characters',
+      validator: password => password.length >= 8
+    },
+  ]
+})
+```
+
+:::info
+Validations always run on `create()`, but only when the `validates` field is present for `update()`. The validation process stops on the first `validator` to return false. If enabled, default validations run first and validate basic properties of both the `'email'` or `'password'` fields.
```

Comment:
```suggestion
Validations always run on `create()`, but only when the field mentioned in `validates` is present for `update()`. The validation process stops on the first `validator` to return false. If enabled, default validations run first and validate basic properties of both the `'email'` or `'password'` fields.
```

Reply:
Will update, thanks!

Reply:
This has been addressed here: https://github.com/wasp-lang/wasp/pull/376/commits/9a545b74ddf9be9e69eff94f13d72b19886f2acc Thanks!